week_3/Binary_Tree/invert_binary_tree/invert_binary_tree.js

```diff
@@ -0,0 +1,28 @@
+// Given the root of a binary tree, invert the tree, and return its root.
+
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val, left, right) {
```

Comment:
what is `TreeNode`?

Reply:
TreeNode is the same thing as the Node class created in index.js. I'll rename it to Node to match index.js.

Reply:
If they're the same, we should rework things so that these files depend on the same type/function. This cuts down on repetition and increases code reliability

Reply:
Should I just import the Node class from `index.js` and get rid of lines 3-10? The algorithm doesn't implement the Node class, but it might serve to demonstrate that it's coming from the same file that the BinarySearchTree class is coming from. (To be clear, the test file doesn't implement the Node class either. It only needs access to the BinarySearchTree class, which implements the Node class in its insert function.)